zipline/zipline_deprecation_warning.py

```diff
@@ -13,20 +13,6 @@
 # See the License for the specific language governing permissions and
```

Comment:
Instead of a having a module dedicated to only a single warning class, what do you think about naming this `warnings.py`, like we have `errors.py`?  Then we can add more later without making the namespace confusing.